src/modules/infobox/components/FeatureListCategoryItem.vue

```diff
@@ -0,0 +1,158 @@
+<script setup>
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { computed, nextTick, ref, toRefs } from 'vue'
+import { useStore } from 'vuex'
+
+import EditableFeature from '@/api/features/EditableFeature.class'
+import LayerFeature from '@/api/features/LayerFeature.class'
+import FeatureDetail from '@/modules/infobox/components/FeatureDetail.vue'
+import ShowGeometryProfileButton from '@/modules/infobox/components/ShowGeometryProfileButton.vue'
+import { canFeatureShowProfile } from '@/store/modules/features.store.js'
+import { FeatureInfoPositions } from '@/store/modules/ui.store.js'
+import { normalizeExtent } from '@/utils/coordinates/coordinateUtils.js'
+import ZoomToExtentButton from '@/utils/ZoomToExtentButton.vue'
+
+const dispatcher = { dispatcher: 'FeatureListCategoryItem.vue' }
+
+const props = defineProps({
+    name: {
+        type: String,
+        required: true,
+    },
+    item: {
+        type: [EditableFeature, LayerFeature],
+        required: true,
+    },
+    showContentByDefault: {
+        type: Boolean,
+        default: false,
+    },
+})
+
+const { name, item, showContentByDefault } = toRefs(props)
+
+const content = ref(null)
+const showContent = ref(!!showContentByDefault.value)
```

Comment:
since we enforce that it should be a `Boolean`, wouldn't it be better to have `showContentByDefault.value`

Reply:
The goal here is to snapshot the value, but then forget about `showContentByDefault` below (if the ref `showContentByDefault` is later changed, `showContent` is unaffected)